src/Microsoft.Data/StringColumn.cs

```diff
@@ -18,7 +16,17 @@ public partial class StringColumn : BaseColumn
 
         public StringColumn(string name, long length) : base(name, length, typeof(string))
         {
-            _stringBuffers.Add(new List<string>());
+            int numberOfBuffersRequired = Math.Max((int)(length / int.MaxValue), 1);
+            for (int i = 0; i < numberOfBuffersRequired; i++)
+            {
+                long bufferLen = length - _stringBuffers.Count * int.MaxValue;
+                List<string> buffer = new List<string>((int)Math.Min(int.MaxValue, bufferLen));
+                _stringBuffers.Add(buffer);
+                for (int j = 0; j < bufferLen; j++)
+                {
+                    buffer.Add(default);
```

Comment:
Is this necessary? If yes - is this the most performant way of appending `bufferLen` `nulls` to the list?

Reply:
Yes for the necessary question. This seemed more efficient to me than appending and doubling the size everytime? 
I don't see a more performant way to appending nulls to a list. If I had an array, I could've just initialized it. I could use LINQ, but I'm not so sure that that is more performant?